functional/MockitoTests/src/test/java/Segmentation.java

```diff
@@ -0,0 +1,13 @@
+package test.java;
+
+import java.util.ArrayList;
+import org.mockito.Mockito;
+
+public class Segmentation {
+    public static void main(String[] args) {
+        System.out.println("This is to trigger a SegmentationError!");
```

Comment:
- This message is misleading. This test does not trigger SegmentationError. We had the segmentation error before when running Mockito.mock().

Please change the message to `System.out.println("Call Mockito.mock(ArrayList.class)");`
- Please rename class `Segmentation` to `MockitoMockTest`